_toc.yml

```diff
@@ -29,27 +29,31 @@ parts:
 - caption: McStas
   chapters:
   - file: 3-mcstas/Simulation
-  - file: 3-mcstas/mcstas-sans
+  # - file: 3-mcstas/mcstas-powder
```

Comment:
I think if you move the notebooks in the config, then you should be able to leave those in the toc?